src/test/test.js

```diff
@@ -0,0 +1,24 @@
+import chai from 'chai';
```

Comment:
Folders should be plural, instead of `test` change it to `tests`

Reply:
Done